### PR TITLE
Update discord markdown pages to reflect recent changes

### PR DESCRIPTION
**Pull request by Beyley**
Removes references to spycrab, adds dev-other, adds Q/A for PPSSPP, and adding PSP guide to how-to-connect

### CODE REVIEW

File: Docs/topics/discord/faq.md [hunk 1]
```diff
@@ -26,3 +26,7 @@ It was born out of frustration when LBP kept violating several internet standard
 Refresher is the patcher that lets you modify your game to connect to custom servers. It can technically work for other games than just the LBP series, but it is still a work in progress.
 
 It aims to be smart, communicating with the server for proper patching instructions via our "AutoDiscover" protocol.
+
+## Q: Is PPSSPP supported by Refresh
```
Comment (jvyden) [inline Docs/topics/discord/faq.md]:
```suggestion
## Q: Is PPSSPP supported by Refresh?
```

File: Docs/topics/discord/dev-info.md [hunk 2]
```diff
@@ -21,6 +21,6 @@ For Refresh, this is accessible at <https://github.com/LittleBigRefresh/Refresh/
 
 <#1102426423240691712> is a channel containing logs from all repositories on our GitHub organization. It contains things like issues, commits, pull requests, and more.
 
-<#1049225857350254632> is the main development channel. Here, you will find development for most of our core projects, mostly Refresh and Bunkum.
+<#1049225857350254632> is the main development channel. Here, you will find development for most of our core projects, mostly [Refresh](<https://github.com/LittleBigRefresh/Refresh>) and [Bunkum](<https://github.com/LittleBigRefresh/Bunkum>).
 
-<#1094447785090236547> is a channel for development of private servers for EA's port of TF2 for the PS3.
+<#1145027319711412284> is the development channel for projects such as [Refresher](<https://github.com/LittleBigRefresh/Refresher>), [Allefresher](<https://github.com/LittleBigRefresh/Allefresher>), and [FreshPresence](<https://github.com/LittleBigRefresh/FreshPresence>)
```
Comment (jvyden) [inline Docs/topics/discord/dev-info.md]:
```suggestion
<#1145027319711412284> is a development channel for projects without a dedicated channel, such as [Refresher](<https://github.com/LittleBigRefresh/Refresher>), [Allefresher](<https://github.com/LittleBigRefresh/Allefresher>), and [FreshPresence](<https://github.com/LittleBigRefresh/FreshPresence>).
```

File: Docs/topics/discord/how-to-connect.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ channelId: 1144875410786623568
 
 # Server Information
 
-LittleBigRefresh currently supports LBP1, LBP2, and LBP Vita. LBP3 is currently experimental, requiring extra steps to work.
+LittleBigRefresh currently supports LBP1, LBP2, LBP Vita, and LBP PSP. LBP3 is currently experimental, requiring extra steps to work.
```
Comment (jvyden) [inline Docs/topics/discord/how-to-connect.md]: I think LBP3 should be fine now, no?

**Review by jvyden — CHANGES_REQUESTED** [review]

File: Docs/topics/discord/how-to-connect.md (hunk 3 above)
Comment (Beyley) [reply Docs/topics/discord/how-to-connect.md]: In general the base gameplay works now yeah, i'll reword it to remove the "experimental" tag
